extension-manifest-v3/test/store/setup.js

```diff
@@ -0,0 +1,4 @@
+import './pre-setup.js';
+import browser from 'webextension-polyfill';
```

Comment:
`webextension-polyfill` is needed as `sinon-chrome` does not support promise based APIs